prac_09/silver_service_taxi.py

```diff
@@ -0,0 +1,20 @@
+from prac_09.taxi import Taxi
+
+class SilverServiceTaxi(Taxi):
+

```

Comment:
It seems your class header and methods are missing docstrings? Perhaps you were busy with other things and forgot?